src/styles/blocks/watch.scss

```
@@ -0,0 +1,46 @@
+.watch {
+  display: flex;
+  margin: auto;
+  height: $watch-size;
+  width: $watch-size;
+  background-color: seashell;
+
+  &__minute-hand {
+    @include watch-parts-position {
+      top: 25%;
+    }
+    height: calc(#{$watch-size} * 0.25);
+    width: 5px;
+    background-color: rgb(201, 76, 76);
+    border-radius: $r-border;
+
+    @include animation-props {
+      animation-duration: 3600s;
+    }
```

no need, you can specify it simply (all places)

 ```suggestion
    animation-duration: 3600s;
```